commanddelegator/command_delegator_api.go

```diff
@@ -175,8 +175,15 @@ func InitEndpoints(r *mux.Router) {
 
 func main() {
 	dbOperator = db.New(starter.GetDbClientConfig())
-	kclient := kubernetesclient.New()
```

Comment:
You should still be creating the Kubernetes client normally as before, it should just be the TLS manager that is subject to change if statement should only wrap around tlsmanager.